src/mapping/common.ts

```diff
@@ -609,18 +613,107 @@ export function _handleBorrowed(
   updateProtocolPosition(protocol, market);
 }
 
+function updateLastP2PIndexesUpdatedInvariant(
```

Comment:
This function must be refactored:
- use `initializers` file to define the default values 
- create an helper to update the different values

Reply:
The "common" file is already really large